components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/listener/IdentityProviderMgtListener.java

```diff
@@ -106,12 +106,25 @@ default boolean doPreDeleteResidentIdpProperties(String tenantDomain,  List<Stri
     /**
      * Define any additional actions after deleting resident idp properties.
      *
-     * @param tenantDomain Tenant domain of the resident idp.
      * @param propertyNames List of property names deleted.
+     * @param tenantDomain Tenant domain of the resident idp.
      * @return Whether execution of this method of the underlying UserStoreManager must happen.
      * @throws IdentityProviderManagementException When an error occurs while handling the event.
      */
-    default boolean doPostDeleteResidentIdpProperties(String tenantDomain, List<String> propertyNames) throws
+    default boolean doPostDeleteResidentIdpProperties(List<String> propertyNames, String tenantDomain) throws
```

Comment:
Were these two methods defined by recently with the same RnD effort and we can ensure no breaking change occur interms of usages ?